### PR TITLE
Update opBNB

### PR DESCRIPTION
**Pull request by ezynda3**
# Which Jira task belongs to this PR?

# Why did I implement it this way?

# Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] This pull request is as small as possible and only tackles one problem
- [ ] I have added tests that cover the functionality / test the bug
- [ ] I have updated any required documentation

# Checklist for reviewer (DO NOT DEPLOY and contracts BEFORE CHECKING THIS!!!)

- [ ] I have checked that any arbitrary calls to external contracts are validated and or restricted
- [ ] I have checked that any privileged calls (i.e. storage modifications) are validated and or restricted
- [ ] I have ensured that any new contracts have had AT A MINIMUM 1 preliminary audit conducted on <date> by <company/auditor>

### CODE REVIEW

File: script/utils/viemScriptHelpers.ts [hunk 1]
```diff
@@ -1,27 +1,49 @@
...
+    id: network.chainId,
+    name: network.name,
+    nativeCurrency: {
+      decimals: 18,
+      name: network.nativeCurrency,
+      symbol: network.nativeCurrency,
+    },
+    rpcUrls: {
+      default: {
+        http: [network.rpcUrl],
```
[the remark this answers is not in the document]
Comment (ezynda3) [reply script/utils/viemScriptHelpers.ts]: not needed

File: config/networks.json [hunk 2]
```diff
@@ -373,6 +373,23 @@
...
+    "status": "active",
+    "type": "mainnet",
+    "rpcUrl": "https://opbnb-mainnet-rpc.bnbchain.org",
+    "verificationType": "etherscan",
+    "explorerUrl": "https://opbnb.bscscan.com",
+    "explorerApiUrl": "https://api-opbnb.bscscan.com/api",
+    "multicallAddress": "0xcA11bde05977b3631167028862bE2a173976CA11",
+    "safeApiUrl": "https://safe-transaction-opbnb-mainnet.bnbchain.org/api",
+    "safeAddress": "0xaAD94196680Edb94B421bb5B1E1A0B27db686C72",
+    "gasZipChainId": 59
```
Comment (0xDEnYO) [inline config/networks.json:391]: this ID is wrong (58 is correct)

Comment (0xDEnYO) [inline config/networks.json:391]: this Id is wrong (should be 58 according to https://dev.gas.zip/gas/chain-support/outbound)

**Review by 0xDEnYO — CHANGES_REQUESTED** [review]

**Review by 0xDEnYO — APPROVED** [review]

File: deployments/opbnb.diamond.json [hunk 3]
```diff
@@ -2,52 +2,52 @@
...
       "0x8Fb506A8040243c1Bc9AC779cF311839176692c3": {
-        "Name": "",
-        "Version": ""
+        "Name": "StandardizedCallFacet",
+        "Version": "1.1.0"
       },
       "0x4E1D2308e11C06c93700FfFdD5b658D2d35a1e15": {
         "Name": "GenericSwapFacetV3",
-        "Version": ""
+        "Version": "1.0.1"
```
[the remark this answers is not in the document]
Comment (ezynda3) [reply deployments/opbnb.diamond.json:50]: Fixed

File: deployments/_deployments_log_file.json [hunk 4]
```diff
@@ -513,9 +513,9 @@
...
         "1.0.0": [
           {
-            "ADDRESS": "0xaD50118509eB4c8e3E39a370151B0fD5D5957013",
+            "ADDRESS": "0x7c76E992B0E65e4AFa657f61bf9fF4b9aeBf3F65",
             "OPTIMIZER_RUNS": "1000000",
-            "TIMESTAMP": "2023-09-18 11:28:56",
+            "TIMESTAMP": "2024-10-07 13:20:56",
             "CONSTRUCTOR_ARGS": "0x",
             "SALT": "",
             "VERIFIED": "false"
```
Comment (0xDEnYO) [inline deployments/_deployments_log_file.json:521]: unverified contract

File: deployments/_deployments_log_file.json [hunk 5]
```diff
@@ -10517,12 +10537,12 @@
...
-            "ADDRESS": "0x5741A7FfE7c39Ca175546a54985fA79211290b51",
+            "ADDRESS": "0x9cb7bE51524DA96806EC38a9A7dDC58608650243",
             "OPTIMIZER_RUNS": "1000000",
-            "TIMESTAMP": "2023-09-18 11:38:38",
-            "CONSTRUCTOR_ARGS": "0x00000000000000000000000011f11121df7256c40339393b0fb045321022ce44",
+            "TIMESTAMP": "2024-10-07 13:51:24",
+            "CONSTRUCTOR_ARGS": "0x00000000000000000000000011f1022ca6adef6400e5677528a80d49a069c00c",
             "SALT": "",
-            "VERIFIED": "true"
+            "VERIFIED": "false"
```
Comment (0xDEnYO) [inline deployments/_deployments_log_file.json:10545]: unverified contract

File: deployments/_deployments_log_file.json [hunk 6]
```diff
@@ -23452,6 +23554,20 @@
...
+          {
+            "ADDRESS": "0x6140b987d6B51Fd75b66C3B07733Beb5167c42fc",
+            "OPTIMIZER_RUNS": "1000000",
+            "TIMESTAMP": "2024-10-07 14:15:11",
+            "CONSTRUCTOR_ARGS": "0x00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000040000000000000000000000000000000000000000000000000000000000000000100000000000000000000000029dacdf7ccadf4ee67c923b4c22255a4b2494ed7",
+            "SALT": "",
+            "VERIFIED": "true"
+          }
+        ]
+      }
```
Comment (0xDEnYO) [inline deployments/_deployments_log_file.json:23570]: why is there a cronos deployment update on this PR? maybe something went wrong here?

**Review by 0xDEnYO — DISMISSED** [review]

File: deployments/_deployments_log_file.json (hunk 5 above)
Comment (ezynda3) [reply deployments/_deployments_log_file.json]: fixed

File: deployments/_deployments_log_file.json (hunk 4 above)
Comment (ezynda3) [reply deployments/_deployments_log_file.json]: fixed

File: deployments/_deployments_log_file.json (hunk 6 above)
Comment (ezynda3) [reply deployments/_deployments_log_file.json:23570]: fixed

File: deployments/_deployments_log_file.json [hunk 7]
```diff
@@ -10517,10 +10537,10 @@
       "production": {
         "1.0.0": [
           {
-            "ADDRESS": "0x5741A7FfE7c39Ca175546a54985fA79211290b51",
+            "ADDRESS": "0x9B112948F3c71eBFb59961657b37c21328cFb01e",
             "OPTIMIZER_RUNS": "1000000",
-            "TIMESTAMP": "2023-09-18 11:38:38",
-            "CONSTRUCTOR_ARGS": "0x00000000000000000000000011f11121df7256c40339393b0fb045321022ce44",
+            "TIMESTAMP": "2024-11-21 11:05:16",
+            "CONSTRUCTOR_ARGS": "0x00000000000000000000000011f1022ca6adef6400e5677528a80d49a069c00c",
```
[the remark this answers is not in the document]
Comment (ezynda3) [reply deployments/_deployments_log_file.json:10543]: No it's not

File: deployments/_deployments_log_file.json (hunk 6 above)
[the remark this answers is not in the document]
Comment (ezynda3) [reply deployments/_deployments_log_file.json]: removed

**Pull request merged**